lib/marshalingCommands.js

```diff
@@ -0,0 +1,173 @@
+var _ = require('lodash');
+var unexpectedItemHandler = require('./unexpectedItemHandler');
+
+/**
+ * Converts boolean value to DynamoDb "BOOL"
+ *
+ * @param item
+ * @returns {*}
+ */
+function marshalBoolean(item) {
+  if (!_.isBoolean(item)) {
```

Comment:
That's pretty cool stuff, feel free to branch off of this and play with those changes. For this pull request probably just going to keep this as a quick port of what we had minus all the babel baggage.